app/src/main/java/com/example/vybe/VibeEvent.java

```diff
@@ -65,6 +67,10 @@ public void setSocialSituation(String socialSituation) {
 
     public void setId(String id) { this.id = id; }
 
+    public String getImage() { return image; }
```

Comment:
this should be called "getPath" to the image. we can do this in our later and outstanding refactoring issue(s) I guess